code/style.css

```diff
@@ -1,58 +1,140 @@
+* { 
+  box-sizing: border-box;
+ }
+ *:before,
+ *:after {
+  box-sizing: border-box;
+ }
+
+html,
+body {
+ height: 100%;
+ position: relative;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: GabrielWeissFriendsFont;
+  background-image: url(./assets/background.jpg);
+  background-size: cover;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+.main-container {
+  min-height: 100vh; 
```

Comment:
what's vh? 😁

Reply:
https://stackoverflow.com/questions/41793228/min-height-in-vh-vs-for-body/41793897